magic-entrypoint.py

```diff
@@ -9,8 +9,9 @@
 
 logging.root.setLevel(logging.INFO)
 
-LISTENS = os.environ["LISTEN"].split()
-NAMESERVERS = os.environ["NAMESERVERS"].split()
+LISTENS            = os.environ["LISTEN"].split()
+NAMESERVERS        = os.environ["NAMESERVERS"].split()
+
```

Comment:
Please could you revert this diff? It's not needed anymore

Reply:
Fixed, sorry about that.